pkg/apis/apps.kubermatic/v1/application_definition.go

```diff
@@ -192,9 +194,15 @@ type ApplicationDefinitionSpec struct {
 	Method TemplateMethod `json:"method"`
 
 	// DefaultValues describe overrides for manifest-rendering in UI when creating an application.
+	// Deprecated: use DefaultValuesBlock instead
 	// +kubebuilder:pruning:PreserveUnknownFields
 	DefaultValues *runtime.RawExtension `json:"defaultValues,omitempty"`
 
+	// DefaultValuesBlock describe overrides for manifest-rendering in UI when creating an application.
+	// Preserves yaml comments.
+	// +kubebuilder:pruning:PreserveUnknownFields
```

Comment:
I don't think this is required here, since there are no unknown fields to preserve.

Reply:
good spot. Removed it.